runtime/syntax/vim.vim

```diff
@@ -5,6 +5,8 @@
 " Version:	7.4-45
 " Automatically generated keyword lists: {{{1
```

Comment:
This comment seems to be wrong, but is also in Vim and I've kept it for easier diff-offsetting.